### PR TITLE
Add workflow for runner_v2 tests

### PR DESCRIPTION
**Pull request by Polber**
Needed to test staging runner v2 worker images against Template ITs

Tests are run against latest release tag for repo to avoid running against broken tests as much as possible

### CODE REVIEW

**Comment by Polber** [comment]
Test failures are unrelated - This PR does not affect existing tests

**Review by damccorm — DISMISSED** [review]
One suggestion, otherwise LGTM

File: .github/workflows/uw-tests.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      - name: Run Integration Tests
+        run: |
+          ./cicd/run-it-tests \
+          --it-region="us-central1" \
+          --it-project="cloud-teleport-testing" \
+          --it-artifact-bucket="cloud-teleport-testing-it-gitactions" \
+          --it-private-connectivity="datastream-private-connect-us-central1" \
+          --it-spanner-host="https://batch-spanner.googleapis.com" \
+          --it-release=true \
+          --it-runner-harness-container-image=$HARNESS_IMAGE \
```
[the remark this answers is not in the document]
Comment (Polber) [reply .github/workflows/uw-tests.yml]: Done

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**